manifests/mod/dav_svn.pp

```diff
@@ -13,7 +13,7 @@
 
   ::apache::mod { 'dav_svn': }
 
-  if $::osfamily == 'Debian' and ($::operatingsystemmajrelease != '6' and $::operatingsystemmajrelease != '10.04' and $::operatingsystemrelease != '10.04' and $::operatingsystemmajrelease != '16.04') {
+  if $::osfamily == 'Debian' and ($::operatingsystemmajrelease != '6' and !($::operatingsystemmajrelease in ['16.04', '18.04'])) {
```

Comment:
The `$::operatingsystemmajrelease != '6'` looks redundant. I hope Debian 6 isn't supported. It's also pretty hard to parse. So the result is that for Ubuntu 14.04 it's `undef` but everything else has `dav_svn_authz_svn.load`?

Reply:
cleaned it up